pkg/generic/generic_service.go

```diff
@@ -107,36 +133,48 @@ func (g *Args) GetOrSetBase() interface{} {
 }
 
 // Write ...
-func (g *Args) Write(ctx context.Context, out thrift.TProtocol) error {
```

Comment:
This seems to be a BreakChange?

Reply:
Actually, yes, but I think there's no better way to pass method name and isClient to the codec of each generic...